internal/ibmcloud-janitor/resources/types.go

```diff
@@ -47,6 +47,7 @@ var VpcResources = []Resource{
 	VPCInstance{},
 	VPCNetwork{},
 	VPCs{},
+	VPCLoadBalancer{},
```

Comment:
check the order, I believe loadbalancers will have subnets and need to be deleted before cleaning up the networks

need your help to review this PR further @dharaneeshvrd 
/cc @Prajyot-Parab

Reply:
Under subnets we will have load balancer as an attached resource, need to cleanup load balancer, before proceeding the deletion of subnet and then vpc.

Reply:
Changed it.